integration/tests/crud/creategetdelete_test.go

```diff
@@ -482,19 +486,27 @@ var _ = Describe("(Integration) Create, Get, Scale & Delete", func() {
 					"--name", mngNG1,
 					"-o", "yaml",
 				)
-				Expect(getMngNgCmd).To(RunSuccessfullyWithOutputString(ContainSubstring("MaxSize: 5")))
-				Expect(getMngNgCmd).To(RunSuccessfullyWithOutputString(ContainSubstring("MinSize: 5")))
-				Expect(getMngNgCmd).To(RunSuccessfullyWithOutputString(ContainSubstring("DesiredCapacity: 5")))
+
+				Expect(getMngNgCmd).To(RunSuccessfullyWithOutputStringLines(
+					ContainElement(HaveSuffix("MaxSize: 5"))),
+					ContainElement(HaveSuffix("MinSize: 5")),
+					ContainElement(HaveSuffix("DesiredCapacity: 5")),
+					ContainElement(HaveSuffix("Type: managed")),
+				)
 
 				getUnmNgCmd := params.EksctlGetCmd.WithArgs(
 					"nodegroup",
 					"--cluster", params.ClusterName,
 					"--name", unmNG1,
 					"-o", "yaml",
 				)
-				Expect(getUnmNgCmd).To(RunSuccessfullyWithOutputString(ContainSubstring("MaxSize: 5")))
-				Expect(getUnmNgCmd).To(RunSuccessfullyWithOutputString(ContainSubstring("MinSize: 5")))
-				Expect(getUnmNgCmd).To(RunSuccessfullyWithOutputString(ContainSubstring("DesiredCapacity: 5")))
+
+				Expect(getUnmNgCmd).To(RunSuccessfullyWithOutputStringLines(
+					ContainElement(HaveSuffix("MaxSize: 5"))),
+					ContainElement(HaveSuffix("MinSize: 5")),
+					ContainElement(HaveSuffix("DesiredCapacity: 5")),
+					ContainElement(HaveSuffix("Type: unmanaged")),
```

Comment:
Writing `unmanaged` in the output might be confusing to users since EKS calls them `self-managed`, but on the other hand, our docs call them `unmanaged` everywhere so it's too late anyway, right? 🤷‍♀️

Reply:
We can ignore this for now since it's not something to be addressed by this PR anyway :)

Reply:
>but on the other hand, our docs call them unmanaged everywhere so it's too late anyway, right? :woman_shrugging:

Yeah, that's true.